code/modules/vore/fluffstuff/guns/protector.dm

```diff
@@ -1,12 +1,12 @@
 // -------------- Protector -------------
 /obj/item/weapon/gun/energy/protector
 	name = "small energy gun"
-	desc = "The KHI-98a 'Protector' is the first firearm custom-designed for Nanotrasen by KHI. It features a powerful stun mode, and \
+	desc = "The HI-98a 'Protector' is the first firearm custom-designed for Nanotrasen by KHI. It features a powerful stun mode, and \
```

Comment:
desc = "The HI-98a 'Protector' is the first firearm custom-designed for Nanotrasen by **>>>KHI<<<**. It features a powerful stun mode, and \

Reply:
good catch, thanks